client/stylesheets/_toCat.scss

```diff
@@ -3802,12 +3802,13 @@ On click the images will be displayed at normal size to complete the effect
     padding: 10px 10px;
     background-color: #f8504b;
     text-align: center;
+    border-top: 3px solid #ee4641;
```

Comment:
Color literals like `#ee4641` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces